packages/widget-markdown/src/extension.ts

```diff
@@ -119,9 +140,30 @@ export function activate (
               stateManager.loadMarkdownContent(selectedDoc)
             }
           )
+          tangle.listen(
+            'externalMarkdownFiles',
+            (externalMarkdownFiles) => {
+              console.log({externalMarkdownFiles})
+            }
+          )
```

Comment:
Looks like this does not listen to configuration changes. It will not be fired and to be updated when settings are updated.

Reply:
Did this now instead: https://github.com/stateful/vscode-marquee/pull/90/files#diff-71345bf3090069793fdb6a0e464c65a831cb92854ddb78cc30085ebbcc1750faR57-R85